### ISSUE TITLE
Column name 'text' breaks sqlalchemy.text() within table class

### ISSUE THREAD

**Issue by rflynn** [rflynn/sqlacodegen#3]
This generated class looks sane:

``` py
from sqlalchemy import Boolean, Column, Integer, text

class T(Base):
    __tablename__ = 't'

    id = Column(Integer, primary_key=True)
    text = Column(Text, nullable=False)
    enabled = Column(Boolean, nullable=False, server_default=text("true")) # calls text()
```

...but generates the following warning:

``` sh
$ python models_public.py
Traceback (most recent call last):
  File "models_public.py", line 4687, in <module>
    class T(Base):
  File "models_public.py", line 4693, in T
    enabled = Column(Boolean, nullable=False, server_default=text("true"))
TypeError: 'Column' object is not callable
```

reversing the order of the 'text' and 'enabled' columns fixes the problem

**Comment by nachtwerk** [comment]
I second this, workaround I use is directly calling sqlalchemy.text()

**Comment by adamatan** [comment]
@nachtwerk @rflynn 
Any idea why `Column` is callable for the `id` field?

**Comment by runout-at** [comment]
same here. i found two workarounds:
if i put the column named 'text' to the end it works.
renaming the column 'text' to eg 'infotext' solves the problem too.
the error appears on the 'mod_timestamp' column.

gives error:
```
class TPersoninfo(Base):
    __tablename__ = 't_personinfo'
    __table_args__ = (
        Index('unique1', 't_person_idt_person', 't_personinfotypes_idt_personinfotypes', unique=True),
    )

    idt_personinfo = Column(Integer, primary_key=True)
    t_person_idt_person = Column(ForeignKey(u't_person.idt_person'), nullable=False, index=True)
    text = Column(String(255), server_default=text("''"))
    t_personinfotypes_idt_personinfotypes = Column(ForeignKey(u't_personinfotypes.idt_personinfotypes'), nullable=False, index=True)
    ins_timestamp = Column(DateTime)
    t_person_idt_insuser = Column(ForeignKey(u't_person.idt_person'), index=True)
    mod_timestamp = Column(DateTime, nullable=False, index=True, server_default=text("CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP"))
    t_person_idt_moduser = Column(ForeignKey(u't_person.idt_person'), index=True)
```

works:
```
class TPersoninfo(Base):
    __tablename__ = 't_personinfo'
    __table_args__ = (
        Index('unique1', 't_person_idt_person', 't_personinfotypes_idt_personinfotypes', unique=True),
    )

    idt_personinfo = Column(Integer, primary_key=True)
    t_person_idt_person = Column(ForeignKey(u't_person.idt_person'), nullable=False, index=True)
    t_personinfotypes_idt_personinfotypes = Column(ForeignKey(u't_personinfotypes.idt_personinfotypes'), nullable=False, index=True)
    ins_timestamp = Column(DateTime)
    t_person_idt_insuser = Column(ForeignKey(u't_person.idt_person'), index=True)
    mod_timestamp = Column(DateTime, nullable=False, index=True, server_default=text("CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP"))
    t_person_idt_moduser = Column(ForeignKey(u't_person.idt_person'), index=True)
    text = Column(String(255), server_default=text("''"))
```

**Comment by pavelkim** [comment]
hey everyone, 

having the same issue:
https://github.com/sqlalchemy/sqlalchemy/issues/5216

**Comment by rotcx** [comment]
> same here. i found two workarounds:
> if i put the column named 'text' to the end it works.
> renaming the column 'text' to eg 'infotext' solves the problem too.
> the error appears on the 'mod_timestamp' column.
> 
> gives error:
> 
> ```
> class TPersoninfo(Base):
>     __tablename__ = 't_personinfo'
>     __table_args__ = (
>         Index('unique1', 't_person_idt_person', 't_personinfotypes_idt_personinfotypes', unique=True),
>     )
> 
>     idt_personinfo = Column(Integer, primary_key=True)
>     t_person_idt_person = Column(ForeignKey(u't_person.idt_person'), nullable=False, index=True)
>     text = Column(String(255), server_default=text("''"))
>     t_personinfotypes_idt_personinfotypes = Column(ForeignKey(u't_personinfotypes.idt_personinfotypes'), nullable=False, index=True)
>     ins_timestamp = Column(DateTime)
>     t_person_idt_insuser = Column(ForeignKey(u't_person.idt_person'), index=True)
>     mod_timestamp = Column(DateTime, nullable=False, index=True, server_default=text("CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP"))
>     t_person_idt_moduser = Column(ForeignKey(u't_person.idt_person'), index=True)
> ```
> 
> works:
> 
> ```
> class TPersoninfo(Base):
>     __tablename__ = 't_personinfo'
>     __table_args__ = (
>         Index('unique1', 't_person_idt_person', 't_personinfotypes_idt_personinfotypes', unique=True),
>     )
> 
>     idt_personinfo = Column(Integer, primary_key=True)
>     t_person_idt_person = Column(ForeignKey(u't_person.idt_person'), nullable=False, index=True)
>     t_personinfotypes_idt_personinfotypes = Column(ForeignKey(u't_personinfotypes.idt_personinfotypes'), nullable=False, index=True)
>     ins_timestamp = Column(DateTime)
>     t_person_idt_insuser = Column(ForeignKey(u't_person.idt_person'), index=True)
>     mod_timestamp = Column(DateTime, nullable=False, index=True, server_default=text("CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP"))
>     t_person_idt_moduser = Column(ForeignKey(u't_person.idt_person'), index=True)
>     text = Column(String(255), server_default=text("''"))
> ```

but why...

**Comment by runout-at** [comment]
maybe (just a guess without digging into the code) because _text_ is a keyword in mysql and should be quoted in a proper way if it is used as a column name.

but it is strange, because it works if it is the last field in the table.

**Comment by ongopongo** [comment]
I filed a ticket in `agronholm`'s repo that I think explains the root cause and includes a general solution: https://github.com/agronholm/sqlacodegen/issues/114

**tl;dr:** - By the time `text` is invoked, it is referring to the class attribute instead of the imported function. The class attribute reassigns `text` from its original value. That's why renaming the `text` column or moving its definition to the end of the class can affect the symptom.
